tests/app/services/test_process_request_json.py

```diff
@@ -58,3 +58,15 @@ def test_should_remove_raw_filter_fields_that_are_non_text():
     assert_equal(result["filter_lot"], "saas")
     assert_equal(result["filter_freeOption"], False)
     assert_equal("freeOption" in result, False)
+
+
+def test_should_add_parent_category():
+    request = {
+        "serviceCategories": ["Accounts payable"],
+    }
+
+    result = convert_request_json_into_index_json(request)
+    assert_equal(result["filter_serviceCategories"], [
+        "accountspayable",
+        "accountingandfinance",
```

Comment:
Are we indexing the value rather than the label? I thought we might want to index the human-readable label, so that we can easily allow keyword searching on categories - if we decide to do that?

Also I think we are moving away from assert_equal to just using normal assert statements. I see that assert_equal is more consistent with the current tests though, so... your call.